tests/components/xy-plot-tests.js

```diff
@@ -165,3 +166,57 @@ test('testing flexible charts', t => {
   t.notEqual(v.height, '100px', 'flexible vis - height is not 100px');
   t.end();
 });
+
+test('Render two stacked bar series with a non-stacked line series chart', t => {
+  const wrapper = shallow(
```

Comment:
Probably be a little better to use your mixed-bar-chart directly instead of creating a new component